apps/www/components/Nav/DevelopersDropdown.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { ReactNode } from 'react'
```

Comment:
Still needed?

Reply:
Ah, no. Thanks for catching it @gregnr 👍